pkg/build/builder/docker.go

```diff
@@ -413,6 +419,33 @@ func traverseAST(cmd string, node *parser.Node) int {
 	return index
 }
 
+// insertEnvAfterFrom inserts an ENV Dockerfile instruction right after the
+// FROM instruction of the given dockerfile to define the environment variables
```

Comment:
Not saying that its not common, just that it is a bit confusing. Guess the sentence is just too long, so splitting it would make it more clear.
Or better:
`insertEnvAfterFrom inserts an ENV Dockerfile instruction together with environment variables defined in env right after the FROM instruction in the given dockerfile variable.`